transactions/reset-epoch/2023/oct-12/arguments.json

```
@@ -0,0 +1,22 @@
+[
+	{
+		"type":"UInt64",
+		"value":"80"
+	},
+	{
+		"type":"String",
+		"value":"f28339976b22b61917e69fb5eca9eeb8"
+	},
+	{
+		"type":"UInt64",
+		"value":"0"
```

```suggestion
		"value":"100000000"
```
100M (current view is 7M)